python/graphframes/tests.py

```diff
@@ -227,6 +261,18 @@ def test_page_rank(self):
         pr = g.pageRank(resetProb, tol=errorTol)
         self._hasCols(pr, vcols=['id', 'pagerank'], ecols=['src', 'dst', 'weight'])
 
+    def test_parallel_personalized_page_rank(self):
+        if GraphFrameTestUtils.is_later_version("2.1"):
```

Comment:
I see - we could define an environment variable like we have for SCALA_VERSION

https://github.com/graphframes/graphframes/blob/master/python/run-tests.sh#L47

but agreed it might not be worthwhile for the effort